### ISSUE TITLE
Support NixOS

### ISSUE THREAD

**Issue by sumnerevans** [navidrome/navidrome#209]
I'd like to be able to deploy Navidrome on NixOS.

**Comment by deluan** [comment]
Seems like an unsung hero is working on that: https://github.com/NixOS/nixpkgs/pull/88928

**Comment by aciceri** [comment]
I created a pull request on nixpkgs for the service: NixOS/nixpkgs#88928

**Comment by deluan** [comment]
Hey @aciceri , what is the status of the NixOS support? Is it all done? Can we close this?

**Comment by sumnerevans** [comment]
So, the package is in to nixpkgs, but there's no way of configuring the service via NixOS's configuration language. I think this MR is fixing that: https://github.com/NixOS/nixpkgs/pull/92621

**Comment by deluan** [comment]
Hey @aciceri what is the status of this? I see there is a fairly recent version of Navidrome published for NixOS: https://repology.org/project/navidrome/versions

Can we close this issue?

**Comment by sumnerevans** [comment]
I'll close this. The package is reasonably up-to-date, but there is no module yet.

**Comment by aciceri** [comment]
@deluan I stopped using `navidrome` lately but sometimes I check it to update the package. Recently I added support for `aarch64-linux` and these changes broke the automatic updates done by [`r-ryantm`](https://github.com/r-ryantm).

@sumnerevans That PR was closed but someone else (better than me writing Nix) fixed my attempts (and merged) the [module definition](https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/audio/navidrome.nix). The `navidrome` json configuration is automatically generated from a Nix attrset.
